moped-database/metadata/tables.yaml

```diff
@@ -5185,6 +5185,147 @@
     - role: moped-viewer
       permission:
         filter: {}
+- table:
+    name: moped_user_saved_views
+    schema: public
+  insert_permissions:
+    - role: moped-admin
+      permission:
+        check: {}
+        set:
+          created_by_user_id: x-hasura-x-hasura-user-db-id
+          updated_by_user_id: x-hasura-x-hasura-user-db-id
+        columns:
+          - description
+          - is_deleted
+          - query_filters
+          - url
+      comment: ""
+    - role: moped-editor
+      permission:
+        check: {}
+        set:
+          created_by_user_id: x-hasura-x-hasura-user-db-id
+          updated_by_user_id: x-hasura-x-hasura-user-db-id
+        columns:
+          - description
+          - is_deleted
+          - query_filters
+          - url
+      comment: ""
+  select_permissions:
+    - role: moped-admin
+      permission:
+        columns:
+          - is_deleted
+          - id
+          - query_filters
+          - description
+          - url
+          - updated_at
+          - created_at
+          - created_by_user_id
+          - updated_by_user_id
+        filter: {}
+      comment: ""
+    - role: moped-editor
+      permission:
+        columns:
+          - is_deleted
+          - id
+          - query_filters
+          - description
+          - url
+          - updated_at
+          - created_at
+          - created_by_user_id
+          - updated_by_user_id
+        filter: {}
+      comment: ""
+    - role: moped-viewer
+      permission:
+        columns:
+          - is_deleted
+          - id
+          - query_filters
+          - description
+          - url
+          - updated_at
+          - created_at
+          - created_by_user_id
+          - updated_by_user_id
+        filter: {}
+      comment: ""
+  update_permissions:
+    - role: moped-admin
+      permission:
+        columns:
+          - description
+          - is_deleted
+          - query_filters
+          - url
+        filter: {}
+        check: null
+        set:
+          updated_by_user_id: x-hasura-x-hasura-user-db-id
+      comment: ""
+    - role: moped-editor
+      permission:
+        columns:
+          - description
+          - is_deleted
+          - query_filters
+          - url
+        filter: {}
+        check: null
+        set:
+          updated_by_user_id: x-hasura-x-hasura-user-db-id
+      comment: ""
+  event_triggers:
```

Comment:
@tillyw could you remove this event trigger? I don't imagine us showing saved view activity anywhere (yet!) since our activity log is on the project level. Thanks!

Let me know if you had something else in mind or if I'm missing an important part. 🙏

Reply:
no that makes sense! thank you for pointing that out :)